lib/shopify_app/engine.rb

```diff
@@ -1,5 +1,15 @@
 # frozen_string_literal: true
 module ShopifyApp
+  module RedactJobParams
+    private
+
+    def args_info(job)
+      log_disabled_classes = %w(ShopifyApp::ScripttagsManagerJob ShopifyApp::WebhooksManagerJob)
```

Comment:
Is there a way to define this behaviour in the classes themselves? It seems like it would be easy to forget to add new classes to the list.

Reply:
*maybe* we could add an attribute to check on each class but these jobs have been consistently  alone for 5 years now :D

Reply:
Fair enough 😄 , I just thought that this isn't something we want to forget in the future.